### ISSUE TITLE
Option to prefer reading externally

### ISSUE THREAD

**Issue by nclm** [FreshRSS/FreshRSS#1954]
I prefer to read articles in their original context, as the author intended, rather than in the interface of a RSS reader. The way to do that in FreshRSS is to click on the “globe-pointer” icon, while the main link has some script on for opening the reader. It would be interesting to have a preference to swap these options:

If the user select “Read content externally by default” (with a better copy), the main links are freed from the JS opening the reader, and the “globe-pointer” icon is swapped by a reader icon and gets the reader-opening script applied to it instead.

**Comment by Alkarex** [comment]
Yes indeed. PR welcome. Currently, holding the control key down while clicking will also open the original Web page.

**Comment by math-GH** [comment]
Mouse-Wheel-Click does the same
